frontend/src/app/features/work-packages/routing/partitioned-query-space-page/partitioned-query-space-page.component.ts

```diff
@@ -257,10 +257,7 @@ export class PartitionedQuerySpacePageComponent extends WorkPackagesViewBase imp
   protected inviteModal = InviteUserModalComponent;
 
   openInviteUserModal():void {
-    const inviteModal = this.opModalService.show(this.inviteModal, 'global');
-    inviteModal.closingEvent.subscribe((modal:any) => {
-      console.log('Modal closed!', modal);
-    });
+    this.opModalService.show(this.inviteModal, 'global');
```

Comment:
the `show` method now returns an observable. if it is not subscribed, nothing will happen. So, this invite modal will never show up, I guess.

Reply:
Oh, my bad. the method has opening the modal as a side effect. IMHO this is a bit of a confusing design, as it leads to assumption like mine above. A better way to implement would be, to not execute anything in `show`until the observable is subscribed. In the consumer it would simply change the usage to 

```
this.opModalService.show(this.inviteModal, 'global').subscribe();
```

Reply:
Let's talk about it on Monday. No blocker for merging.